### PR TITLE
Postmortem

### PR DESCRIPTION
**Pull request by sabau**
**Description:**
Description of the problem we faced this morning and the solution
Root cause still not found

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: PENDING.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 ### Added
 
+- incidend POSTMORTEM.md @sabau
```
Comment (fedekunze) [inline PENDING.md]:
```suggestion
- [\#2256](https://github.com/cosmos/voyager/pull/2256) incident `POSTMORTEM.md` @sabau
```

File: docs/POSTMORTEM.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Postmortem summary
```
Comment (faboweb) [inline docs/POSTMORTEM.md]: let's give the postmortem a name and date maybe?

**Comment by faboweb** [comment]
Added it to notion

**Comment by faboweb** [comment]
Thanks for the write up

**Pull request closed without merging**